app/src/main/java/es/wolfi/utils/ColorUtils.java

```diff
@@ -0,0 +1,50 @@
+/**
+ *  Passman Android App
+ *
+ * @copyright Copyright (c) 2017, Andy Scherzinger
+ * @license GNU AGPL version 3 or any later version
```

Comment:
I'll add them, even though technically (as in copyright) this isn't true. But I don't mind :)